platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/RoleClaimsHandler.java

```diff
@@ -207,7 +215,34 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
 
             connection = connectionFactory.getConnection();
             if (connection != null) {
-                BindResult bindResult = connection.bind(bindUserDN, bindUserCredentials.toCharArray());
+                BindRequest request;
+                switch (bindMethod) {
+                case "Simple":
```

Comment:
@stustison Case sensitive admin config options make me shudder. Any reason these need to be case sensitive options (regardless if this is a human typing the value or a wizard). [I'm thinking like the setup of idp.. remembering if you need to do upper or lower IDP is a pain]